test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -184,6 +262,53 @@ public void windowClosing(WindowEvent e) {
         windowList.add(frame);
     }
 
+    private static void captureScreen(Rectangle bounds) {
+        if (robot == null) {
+            try {
+                robot = new Robot();
+            } catch (AWTException e) {
+                String errorMsg = "Failed to create the " +
+                        "instance of Robot.";
+                JOptionPane.showMessageDialog(frame, errorMsg, "Failed",
+                        JOptionPane.ERROR_MESSAGE);
+                forceFail(errorMsg + e.getMessage());
+            }
+        }
+        MultiResolutionImage multiResolutionImage = robot
+                .createMultiResolutionScreenCapture(bounds);
+        List<Image> imageList =
+                multiResolutionImage.getResolutionVariants();
+        Image image = imageList.get(imageList.size() - 1);
+        File file = new java.io.File("CaptureScreen_" +
+                imgCounter.incrementAndGet() + ".png");
+        try {
+            ImageIO.write((RenderedImage) image, "png", file);
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage());
```

Comment:
You shouldn't throw away the original exception, it may be useful in diagnosing the problem.
```suggestion
            throw new RuntimeException(e);
```

Show message to the user? Or, add a try-catch inside `captureScreen` to show this message.